JoonasHeinonen/escape-from-zordoom-prison
# Music from Ratchet & Clank

## [Music from Ratchet & Clank](https://trello.com/c/glWFQrwN/60-music-from-ratchet-clank)

## Changes:
Added the possibility for each level to have a musical track that is played in loop. 

## How to test:
 - This feature can for now, be tested in the Test level, although you're able to add music on any level.
 - Ensure that the music plays on loop.

## Sources cited:
 - [How to create a AudioStreamPlayer in GDScript](https://forum.godotengine.org/t/how-to-create-a-audiostreamplayer-in-gdscript/28037)
 - [AudioStreamPlayer](https://docs.godotengine.org/en/stable/classes/class_audiostreamplayer.html#audiostreamplayer)
 - [AudioStreamMP3](https://docs.godotengine.org/en/stable/classes/class_audiostreammp3.html#audiostreammp3)
